api-reference/10 UI Widgets/dxDataGrid/1 Configuration/export/export.md

```diff
@@ -14,25 +14,39 @@ A user can click the **Export** button to save an Excel file with the exported d
 The following instructions show how to enable and configure client-side export:
 
 1. **Reference the required libraries**  
-    Reference or import the <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> v3.3.1 or newer and <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> libraries. Export also requires the <a href="https://stuk.github.io/jszip/" target="_blank">JSZip</a> library. In a modular environment, this library is listed in package dependencies and is already added. If your app does not use modules, reference JSZip manually.
+    Reference or import the <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> and <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> libraries
 
     ---
     ##### jQuery
         <!--HTML-->
         <head>
             <!-- ... -->
             <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/7.4.0/polyfill.min.js"></script>
-            <script src="https://cdnjs.cloudflare.com/ajax/libs/exceljs/3.3.1/exceljs.min.js"></script>
-            <script src="https://cdnjs.cloudflare.com/ajax/libs/FileSaver.js/1.3.8/FileSaver.min.js"></script>
-            <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.1.5/jszip.min.js"></script>
+            <script src="https://cdnjs.cloudflare.com/ajax/libs/exceljs/4.1.1/exceljs.min.js"></script>
+            <script src="https://cdnjs.cloudflare.com/ajax/libs/FileSaver.js/2.0.2/FileSaver.min.js"></script>
             <!-- reference the DevExtreme sources here -->
         </head>
 
     ##### Angular   
 
+        <!-- tab: Installation command -->
+        npm install --save exceljs@4.1.1 file-saver
+
+        <!-- tab: tsconfig.app.json -->
+        // add ExcelJS to the *"paths"* section:
+        "compilerOptions": {
+            // ...
+            "paths": {
+                // ...
+                "exceljs": [
+                    "node_modules/exceljs/dist/exceljs.js"
+                ]
+            }
+        }
+
         <!-- tab: app.component.ts -->
         import { Component } from '@angular/core';
-        import ExcelJS from 'exceljs';
+        import { Workbook } from 'exceljs';
         import saveAs from 'file-saver';
```

Comment:
I noticed that neither ExcelJS nor FileSaver are used until p.3. Maybe we should wait with showing the imports until that time.